packages/falter-common/files-common/lib/upgrade/keep.d/ffwizard3

```diff
@@ -0,0 +1,2 @@
+# preserve the data of ffwizard3 on a sysupgrade
+/etc/ffwizard3.json
```

Comment:
I would use /etc/ffwizard.json or /etc/ff-config.json. There is no point in adding a number to something that is new.

Reply:
I agree, that adding merely a number has no real point. In this case I consider it something special though.

There is already the legacy ffwizard-format, which is used by the current wizard. I feel, that we should distinguish the new format from the old somehow.

Additionally, there is already a ffwizard2 written by @stargieg. So I thought, it might be a good idea, to name the file after the to be written project for the new ffwizard, which might be ffwizard3 then.